modules/nw-cluster-mtu-change.adoc

```diff
@@ -91,125 +91,14 @@ where:
 `<node_name>`:: Specifies the name of a node in your cluster.
 --
 
-... To find the connection profile that NetworkManager created for the interface name returned from the previous command, enter the following command:
-+
-[source,terminal]
-----
-$ oc debug node/<node_name> -- chroot /host nmcli c | grep <interface>
-----
-+
---
-where:
-
-`<interface>`:: Specifies the name of the primary network interface.
---
-+
-.Example output for OpenShift SDN
-[source,text]
-----
-Wired connection 1  46da4a6a-xxxx-xxxx-xxxx-ac0ca900f213  ethernet  ens3
-----
-+
-.Example output for OVN-Kubernetes without an original connection configuration
-[source,text]
-----
-ovs-if-phys0        353774d3-0d3d-4ada-b14e-cd4d8824e2a8  ethernet       ens4
-ovs-port-phys0      332ef950-b2e5-4991-a0dc-3158977c35ca  ovs-port       ens4
-----
-+
---
-For the OVN-Kubernetes network plugin, two or three connection manager profiles are returned.
-
-* If the previous command returns only two profiles, then you must use a default NetworkManager connection configuration as a template.
-* If the previous command returns three profiles, use the profile that is not named `ovs-if-phys0` or `ovs-port-phys0` as a template for the following modifications.
---
-
-... To get the file name of the NetworkManager connection configuration for the primary network interface, enter the following command:
-+
-[source,terminal]
-----
-$ oc debug node/<node_name> -- chroot /host nmcli -g UUID,FILENAME c show | grep <uuid> | cut -d: -f2
-----
-+
---
-where:
-
-`<node_name>`:: Specifies the name of a node in your cluster.
-`<uuid>`:: Specifies the UUID of the NetworkManager connection profile.
---
-+
-.Example output
-[source,text]
-----
-/run/NetworkManager/system-connections/Wired connection 1.nmconnection
-----
-
-... To copy the NetworkManager connection configuration from the node, enter the following command:
-+
-[source,terminal]
-----
-$ oc debug node/<node_name> -- chroot /host cat "<profile_path>" > config.nmconnection
-----
-+
---
-where:
-
-`<node_name>`:: Specifies the name of a node in your cluster.
-`<profile_path>`:: Specifies the file system path of the NetworkManager connection from the previous step.
---
+... Edit the NetworkManager configuration file saved in the `config.conf` file:
```

Comment:
`config.con` seems redundant and could use a more indicative name. `<interface_name>-mtu.conf` could work better.

Reply:
Addressed